RubberduckBaseProject.csproj

```diff
@@ -17,7 +17,7 @@
     <!-- Ignore MSB4011 warning, rationale above -->
     <!-- Ignore VisualStudio whining about the CodeAnalysis assembly changing (IDE1001) -->
     <DisabledWarnings>$(DisabledWarnings);4011;1001</DisabledWarnings>
-    <Version Condition=" '$(Version)' == ''">2.3.0</Version>
+    <Version Condition=" '$(Version)' == ''">2.3.1</Version>
```

Comment:
this is fine, but you missed line 28 here :)

Reply:
Line 28 didn't have the revision# though. Added.